examples/codeviewer/gradle.properties

```diff
@@ -10,6 +10,6 @@ kotlin.native.useEmbeddableCompilerJar=true
 kotlin.mpp.androidSourceSetLayoutVersion=2
 # Enable kotlin/native experimental memory model
 kotlin.native.binary.memoryModel=experimental
-kotlin.version=2.1.20
-agp.version=8.5.0
-compose.version=1.8.2
+kotlin.version=2.2.10
+agp.version=8.9.0
+compose.version=1.9.0-rc01
```

Comment:
We shouldn't use non-stable version in the examples

Reply:
We should keep 1.8.2, and it should work locally and on the CI with the building version